google_authorized_keys/main.go

```diff
@@ -49,43 +52,14 @@ func logFormatWindows(e logger.LogEntry) string {
 	return fmt.Sprintf("%s %s: %s", now, programName, e.Message)
 }
 
-func getMetadata(key string, recurse bool) ([]byte, error) {
-	client := &http.Client{
-		Timeout: defaultTimeout,
-	}
+func getMetadata(ctx context.Context, key string) ([]byte, error) {
```

Comment:
Is it worth keeping this wrapper?

Reply:
after refactoring it wasn't really doing much. Got rid of it. Thanks!